ouroboros-network/test-cddl/Main.hs

```diff
@@ -57,131 +57,177 @@ import qualified Ouroboros.Network.Protocol.LocalStateQuery.Test as LocalStateQu
 import qualified Ouroboros.Network.Protocol.LocalTxSubmission.Test as LocalTxSubmission (Tx, Reject)
 
 
-cddlTool :: FilePath
-cddlTool = "cddl"
-
-diag2cborTool :: FilePath
-diag2cborTool = "diag2cbor.rb"
-
+-- | The main program, it requires both
+--
+-- - 'cddl' program
+-- - 'diag2cbor.rb' script
+--
+-- to be installed in the '$PATH'.
+--
```

Comment:
I don't think removing `cddlTool` and `diag2cborTool` and hardcoding the program names below is an improvement.

Reply:
They are only used in one place so it's easier to see what happens if they are inlined.